Projekty/Projekt1/AdriannaGrudzień_KingaUłasik/WUM_projekt1_inżynieria_cech.ipynb

```diff
@@ -0,0 +1,1810 @@
+{
```

Comment:
<p>To, że jedna zmienna objaśniająca jest skorelowana z drugą, a nie z tagretem, jest przesłanką za wyrzuceniem jej, a nie za uwzględnianiem</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>